### PR TITLE
MRG: automatic plotting of channel names in evoked.plot_image

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
Addresses another point in #5170

~Todo: tests~

### CODE REVIEW

**Comment by agramfort** [comment]
how does it look like?

**Comment by jona-sassenhagen** [comment]
Ideally, as before:

https://8216-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_stats_cluster_erp.html

https://8216-1301584-gh.circle-artifacts.com/0/html/_images/sphx_glr_plot_stats_cluster_erp_002.png

(with this PR)

It's supposed to just save lines for ERP viz as in the example.

File: mne/evoked.py [hunk 1]
```diff
@@ -311,13 +311,14 @@ def plot_image(self, picks=None, exclude='bads', unit=True, show=True,
...
-                   mask_cmap='Greys', mask_alpha=.25, time_unit='s'):
+                   mask_cmap='Greys', mask_alpha=.25, time_unit='s',
+                   show_ch_names=None):
         return plot_evoked_image(
             self, picks=picks, exclude=exclude, unit=unit, show=show,
             clim=clim, xlim=xlim, proj=proj, units=units, scalings=scalings,
             titles=titles, axes=axes, cmap=cmap, colorbar=colorbar, mask=mask,
             mask_style=mask_style, mask_cmap=mask_cmap, mask_alpha=mask_alpha,
-            time_unit=time_unit)
+            time_unit=time_unit, show_ch_names=show_ch_names)
```
Comment (agramfort) [inline mne/evoked.py]: I see that we call it everywhere show_names and not show_ch_names

`git grep show_names`

**Comment by jona-sassenhagen** [comment]
I think it’s a false positive:

„No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.“

But I get that on restarts too. And the same build errors on my other PRs,
too. 

I already pinged Eric on Gitter ...

**Comment by agramfort** [comment]
@jona-sassenhagen please set PR to MRG when you think they are ready. Thx

**Comment by jona-sassenhagen** [comment]
The test kinda sucks ... I want to see if I can come up with a better test.

**Comment by jona-sassenhagen** [comment]
Tests look good. Ready for merge from my end.

**Comment by agramfort** [comment]
thx @jona-sassenhagen

**Comment by jona-sassenhagen** [comment]
:)

**Pull request merged**